### DISCUSSION TITLE
Forward node to ingest logs?

### DISCUSSION THREAD

**Discussion by OzzyKampha** [Security-Onion-Solutions/securityonion#4709] [Unsupported Versions]
Can logs get send to a forward node or does it need to get send to the master

**Comment by weslambert** [comment]
It depends on what/how you are sending.  We typically advise to send to the manager if possible, but if you are sending syslog or to an input that Filebeat supports, then you should be able to send to a sensor assuming the appropriate firewall rules are in place.

**Reply by OzzyKampha** [reply]
Good to know. I was thinking on using a forward node for all logs in a net then only allow communication from forward node to master node in the fw. 
is it only filebeat or winlogbeat as well?

**Reply by OzzyKampha** [reply]
So if i so-allow for beats, it will allow this on the forward node, or do i have to make a custom rule?
